src/components/VmActions/actions.js

```diff
@@ -0,0 +1,20 @@
+import { SET_CONSOLE_IN_USE, CONSOLE_IN_USE } from './constants'
```

Comment:
The idea of dividing is to make every module more independent, and this action use only this module, so there can not be any conflicts.